digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/util/GeometryTransform.scala

```diff
@@ -326,38 +312,27 @@ class VKMGeometryTransform {
     val roadPart = validateAndConvertToInt(VkmRoadPart, data)
     val track = validateAndConvertToInt(VkmTrackCode, data)
     val mValue = validateAndConvertToInt(VkmDistance, data)
-    val deviation = convertToDouble(data.get(VkmOffsetToGivenCoordinates))
+    val deviation = Option(0.0)
```

Comment:
Why is this always 0.0?

Reply:
@oskarmalmstrom in the old version we get this value from json, but now we don't have it

Reply:
If we don't get this now and the case class RoadAddress uses it maybe its better if we take out the parameter of the case class and remove this deviation since this is no longer used.
Code that are using this case class puts None except this one

Reply:
Applied last comment